src/main/java/com/ldtteam/blockui/views/ScrollingListContainer.java

```diff
@@ -30,7 +30,7 @@ public void refreshElementPanes(final ScrollingList.DataProvider dataProvider, f
             for (int i = 0; i < numElements; ++i)
             {
                 final Pane child;
-                final int childYpos = (childSpacing + listElementHeight) * i;
+                final int childYpos = listElementHeight * i;
                 if (childYpos + listElementHeight >= scrollY && childYpos <= scrollY + height)
```

Comment:
```suggestion
                if (childYpos + (listElementHeight - childSpacing) >= scrollY && childYpos <= scrollY + height)
```
since you are here (from discord discussion)